packages/css/src/breadcrumb/breadcrumb.scss

```diff
@@ -0,0 +1,73 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+@import "../../utils/breakpoint";
+
+@mixin reset {
+  box-sizing: border-box;
+  break-after: avoid;
+  break-inside: avoid;
+  margin-block: 0;
+  margin-inline: 0;
```

Comment:
`margin-inline` is 0 by default, right?

Reply:
It is, copied the `reset` from some other file. `<ol>` doesn't need a  `margin-inline: 0` Because that is the default.